README.md

```diff
@@ -56,6 +45,28 @@ See the [documentation](http://pm2.keymetrics.io/docs/usage/docker-pm2-nodejs/#u
 }
 ```
 
+### Create a `Dockerfile` in your Node.js app project
+Assuming that `pm2.json` and `package.json` are in the same folder of the `Dockerfile` and that the `src` folder contains your source code.
+
+```dockerfile
+FROM keymetrics/pm2-docker-alpine:latest
+
+# Bundle APP files
+COPY src src/
+COPY package.json .
+COPY pm2.json .
```

Comment:
I dont think `pm2 process file` need to be between backtick cause people will think its a command to run, by the way we should name it `pm2 ecosystem` because when you run the `pm2 ecosystem` command pm2 will generate a sample file